zipline/utils/calendars/_calendar_helpers.pyx

```diff
@@ -1,11 +1,11 @@
-from numpy cimport ndarray, long_t
-from numpy import searchsorted
+from numpy cimport ndarray, long, long_t
```

Comment:
This probably wants to be `int64_t` for anything that's converting to/from a nanos value.  datetimes/timestamps are always 64-bit in numpy, but the size of `long` is system-dependent.